sdk/translation/azure-ai-translation-document/src/test/java/com/azure/ai/translation/document/DocumentTranslationStatus.java

```diff
@@ -0,0 +1,24 @@
+// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License.
+
+package com.azure.ai.translation.document;
+
+public enum DocumentTranslationStatus {    
```

Comment:
Why do we need this enum in test code? We should use the `Status` enum defined in the source models package.

Reply:
Deleted and updated to use Status model